concepticondata/conceptlists/Lastra-1986-381.tsv

```diff
@@ -0,0 +1,382 @@
+ID	NUMBER	SPANISH	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Lastra-1986-381-1	1	agua	water	948	WATER
+Lastra-1986-381-2	2	fuego	fire	221	FIRE
+Lastra-1986-381-3	3	humo	smoke	778	SMOKE (EXHAUST)
+Lastra-1986-381-4	4	hollín	soot		
+Lastra-1986-381-5	5	ceniza	ash	646	ASH
+Lastra-1986-381-6	6	carbón	coal	1188	CHARCOAL
+Lastra-1986-381-7	7	tierra	land	1228	EARTH (SOIL)
```

Comment:
`3603 EARTH OR LAND` or `626 LAND`?

Reply:
I made a mistake on this one. I'll say the translation should be earth (soil) and the mapping to conception is correct. So, we can change this accordingly on the ENGLISH column.